modulemd/include/modulemd-2.0/modulemd-errors.h

```diff
@@ -45,6 +101,8 @@ G_BEGIN_DECLS
  * library. Since: 2.8
  *
  * Since: 2.0
+ * Deprecated: 2.9
+ * Use #ModulemdError instead.
```

Comment:
Two things I just dug up...

On the gtk-doc doc page explaining the use of "Deprecated:" at https://developer.gnome.org/gtk-doc-manual/stable/documenting_symbols.html.en, the provided examples show the format to use is a single line: "Deprecated: 2.9: Use #ModulemdError instead."

I also noticed there is a `MMD_DEPRECATED_TYPE_FOR` macro defined in `modulemd-deprecated.h`. `MMD_DEPRECATED_TYPE_FOR (ModulemdError)` should be placed below.

Reply:
Of course, that also means `#include "modulemd-deprecated.h"` is needed above.

Reply:
Experimentation shows that gtk-doc also picks up the text on the lines immediately following the "Deprecated:" line and properly includes it in the generated documentation. So there's no need to change that.

But please do add `MMD_DEPRECATED_TYPE_FOR()`. Thanks!

Reply:
It turns out that `MMD_DEPRECATED_TYPE_FOR()` does not work for the enum as a whole, but I can specify it for the individual enumerations within it instead.